### ISSUE TITLE
Add zfs-linux-rt and zfs-linux-rt-lts

### ISSUE THREAD

**Issue by szanni** [archzfs/archzfs#332]
Is there any chance of adding a _zfs-linux-rt_ and/or _zfs-linux-rt-lts_ package for everyone using linux-rt and linux-rt-lts?

I just modified the zfs-linux-lts package from AUR to point to linux-rt (5.4.24.15-1). Seems to work just fine.

**Comment by fryfrog** [comment]
Why not use the `zfs-dkms` package instead? It'll build for what ever kernel your heart desires.

**Comment by szanni** [comment]
I am a little wary with regards to `zfs-dkms`. Why do the `zfs-linux` packages exist in the first place if `zfs-dkms` is sufficient? I do not like taking chances with my file systems - silent corruptions etc.

**Comment by fryfrog** [comment]
Different strokes for different folks. Do you not know what `dkms` is?

The `zfs-linux` package *only* works w/ the `linux` package it was built for. So you *have* to use the right kernel and you *have* to use the right zfs with that kernel. That means when the kernel is updated, you have to *wait* until the zfs package is *also* built. But they're compiled, so installation is simple.

The `dkms` packages on the other hand *build* as part of the kernel upgrade. So you can run *any* kernel, and it'll work. You can update your kernel *as soon as* it comes out, instead of having to wait. But it isn't a pre-compiled package, so it might take 5-10 minutes to build. Or more on a wimpy system.

The end result is the *same* zfs. For you, it means you can use what ever kernel you want. For them, it means not having to support a bunch of non-default kernels.

**Comment by szanni** [comment]
I do understand what `dkms` is although I'm not intimately familiar with the implementation details.
That said: I do not fully trust it - at least with my file system. I have had problems in the past with funky display drivers.

My reluctancy  in this particular case stemmed from the different patches that were applied to `zfs-linux=0.8.3` for each kernel version. Looking at the new PKGBUILD for version 0.8.4 all of these patches seem to have been applied conditionally upstream. So... I might check out the `dkms` driver after all.

**Comment by fryfrog** [comment]
Hold the phone, [linux-rt](https://aur.archlinux.org/packages/linux-rt/) and [linux-rt-lts](https://aur.archlinux.org/packages/linux-rt-lts/) are *AUR* packages, not an official Arch kernel. Does it even make sense for archzfs to cover it? It looks like they even have an unofficial repository. Maybe that'd be a more appropriate place for it?

**Comment by szanni** [comment]
Yes, both are _AUR_ packages. Is that a problem? There is [zfs-linux-vfio](https://aur.archlinux.org/packages/zfs-linux-vfio/) for one, that depends on [linux-vfio](https://aur.archlinux.org/packages/linux-vfio/) -> _AUR_.

**Comment by fryfrog** [comment]
Then why not be the change you want to see in the world and create
equivalent air packages yourself?

**Comment by szanni** [comment]
Host a repository? That is precisely what I did not intend on doing. Setting up the entire build system, VMs & keeping everything up to date is not really realistic for me at this moment.
I guess `zfs-linux-rt` is going to stay just a normal AUR package for the time being. I guess this issue can be closed then?
